recipe/meta.yaml

```diff
@@ -60,14 +68,17 @@ test:
   imports:
     - onnxruntime
   commands:
+    - python -c "import onnxruntime" 
```

Comment:
```suggestion
```
Assuming this doesn't add anything on top of L69?

Reply:
Thanks, it was a debug leftover.